apps/client/src/components/organisms/panel/Panel.component.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactElement, useEffect, useState } from 'react';
+import { Description, Text } from '../..';
+import { ReactComponent as ArrowHeadTop } from 'pixelarticons/svg/chevron-up.svg';
+import cs from 'classnames';
+
+export interface PanelProps {
+  title: string;
+  children: ReactElement<typeof Description>[];
+}
+
+export function Panel({ title, children }: PanelProps): ReactElement {
+  const [isCollapse, setCollapse] = useState(true);
+  useEffect(() => {
+    const element = document.getElementById('scrollPanel');
```

Comment:
Do we only have a single `scrollPanel` element through out the app?

Reply:
only one scroll panel per page